argument.go

```diff
@@ -41,6 +42,10 @@ func (o arg) GetLname() string {
 	return o.lname
 }
 
+func (o arg) IsFlag() bool {
+	return o.size == 1
```

Comment:
`o.size` does not mean it is a flag. flag would be a type of the interface successfully cast to boolean.